edx_exams/settings/devstack.py

```diff
@@ -52,3 +52,9 @@
 })
 
 LMS_ROOT_URL = 'http://edx.devstack.lms:18000'
+
+EVENT_BUS_PRODUCER = 'edx_event_bus_kafka.create_producer'
+EVENT_BUS_CONSUMER = 'edx_event_bus_kafka.KafkaEventConsumer'
+EVENT_BUS_KAFKA_SCHEMA_REGISTRY_URL = 'http://edx.devstack.schema-registry:8081'
+EVENT_BUS_KAFKA_BOOTSTRAP_SERVERS = 'edx.devstack.kafka:29092'
+EVENT_BUS_TOPIC_PREFIX = 'dev'
```

Comment:
Are we also supposed to add a blank value for `EVENT_BUS_PRODUCER_CONFIG` per the plan that Becca gave, or is that supposed to be done when we pushed to prod?

Reply:
I think it makes sense to add here as well. Otherwise, it won't work in development. I'll add it.

We'll have to add all of this configuration to the stage and production configuration files at some point, which is where that setting will really matter.